integration/src/integration/java/pl/allegro/tech/hermes/integration/management/SubscriptionManagementTest.java

```diff
@@ -260,6 +259,124 @@ public void shouldReturnNoDataStatusWhenGraphiteRespondsWithAnError() {
         assertThat(subscriptionHealth).isEqualTo(SubscriptionHealth.NO_DATA);
     }
 
+    @Test
+    public void shouldReturnCorrectResultWhenValidatingJsonFilters() {
+        // given
+        String groupName = "jsonFilters";
+        String topicName = "topic";
+        String subscriptionName = "subscription";
+        AvroUser user = new AvroUser();
+
+        // and
+        Topic topic = operations.buildTopic(groupName, topicName);
+        final MessageFilterSpecification spec1 = new MessageFilterSpecification(of("type", "jsonpath", "path", ".name", "matcher", "defaultName"));
```

Comment:
split this and next test into two so that each response value has it's own specific check
in spock this would be simpler with params but perhaps it's not feasible having the base integration tests all set up already